README.md

```diff
@@ -174,9 +177,9 @@ If you are working for Rapid River on your own computing equipment, it must run
 
 You must configure your computer to apply critical patches from your operating system vendor within 30 days of their release.
 
-Your computing devices should include software to automatically scan for viruses and malware using up-to-date virus and malware signatures. Your computing devices should also employ a software firewall in addition to any local hardware or enterprise firewalls you may have configured.
+Your computing devices should include software to automatically scan for viruses and malware using an up-to-date virus and malware signatures. Your computing devices should also employ a software firewall in addition to any local hardware or enterprise firewalls you may have configured.
```

Comment:
`an up-to-date ... signatures` is incorrect (`signatures` is plural; therefore the article `an` does not match).